paddle/fluid/framework/ir/graph_pattern_detector.cc

```diff
@@ -1004,11 +1003,12 @@ PDNode *patterns::Transpose2Dequant::operator()() {
   auto *transpose2_op =
       pattern->NewNode(transpose2_op_repr())->assert_is_op("transpose2");
   auto dequant_in = pattern->NewNode(dequant_in_repr())
+                        ->AsInput()
+                        ->AsIntermediate()
```

Comment:
AsIntermediate() overwrites the result of AsInput()

Reply:
You're right thanks!